mcs/class/System/Test/System.Net.Sockets/SocketTest.cs

```diff
@@ -3440,6 +3440,7 @@ public void UdpDoubleBind ()
 
 				try {
 					ss.Bind (new IPEndPoint (IPAddress.Any, ep.Port));
+					ss.Listen (1);
```

Comment:
Was this listen enough to make the test fail? Or do we have to do listen on the first socket too?